lightning/src/chain/chainmonitor.rs

```diff
@@ -187,20 +189,23 @@ where
 	P::Target: channelmonitor::Persist<ChannelSigner>,
 {
 	fn transactions_confirmed(&self, header: &BlockHeader, txdata: &TransactionData, height: u32) {
+		log_debug!(self.logger, "{} transactions confirmed at height {} in block {}", txdata.len(), height, header.block_hash());
```

Comment:
Maybe qualify "transactions" with "relevant" so it's clear the block may have had more transactions? Though I suppose the user could pass anything to this so, they may not all be relevant.